docker/server/entrypoint.sh

```diff
@@ -81,6 +106,13 @@ if [ -n "$(ls /docker-entrypoint-initdb.d/)" ]; then
     clickhouseclient=( clickhouse-client --multiquery -u $CLICKHOUSE_USER $WITH_PASSWORD )
```

Comment:
SHouldn't we also add `--database $CLICKHOUSE_DB` here?

Reply:
Good question. I think - no, because another init scripts can work with other databases, and this can breaks backward compatibility with previous images